frontend/apps/filemanager/filemanager.lua

```diff
@@ -359,6 +383,13 @@ function FileManager:pasteHere(file)
         local dest = lfs.attributes(file, "mode") == "directory" and
             file or file:match("(.*/)")
         if self.cutfile then
+            -- if we move a file, also move its sidecar directory
+            if lfs.attributes(orig, "mode") == "file" then
+                local orig_sdr = orig:match("%.") and orig:match("(.*)%.")..".sdr" or nil
```

Comment:
Better use `getSidecarDir` to get the sidecard directory path. You can find examples within this file.